sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/AmqpConnection.java

```diff
@@ -79,4 +80,23 @@
      * @return A stream of shutdown signals that occur in the AMQP endpoint.
      */
     Flux<AmqpShutdownSignal> getShutdownSignals();
+
+    /**
+     * Gets or creates the management node.
+     *
+     * @param entityPath Entity for which to get the management node of.
+     * @return A Mono that completes with the management node.
```

Comment:
I would indicate that this may throw an `UnsupportedOperationException` if this functionality hasn't been implemented by the implementation class.